### ISSUE TITLE
isAbstract

### ISSUE THREAD

**Issue by aerik2** [MichalLytek/typegraphql-prisma#391]
Seems like this was removed here: https://github.com/MichalLytek/type-graphql/discussions/1455

But it's still being generated:

```
src/generated/graphql/resolvers/outputs/WellIdentifierMaxAggregate.ts:7:3 - error TS2345: Argument of type '{ isAbstract: boolean; }' is not assignable to parameter of type 'ObjectTypeOptions'.
  Object literal may only specify known properties, and 'isAbstract' does not exist in type 'ObjectTypeOptions'.

7   isAbstract: true
    ~~~~~~~~~~~~~~~~

src/generated/graphql/resolvers/outputs/WellIdentifierMinAggregate.ts:7:3 - error TS2345: Argument of type '{ isAbstract: boolean; }' is not assignable to parameter of type 'ObjectTypeOptions'.
  Object literal may only specify known properties, and 'isAbstract' does not exist in type 'ObjectTypeOptions'.

7   isAbstract: true
    ~~~~~~~~~~~~~~~~

src/generated/graphql/resolvers/outputs/WellIdentifiersOnOrdersCountAggregate.ts:7:3 - error TS2345: Argument of type '{ isAbstract: boolean; }' is not assignable to parameter of type 'ObjectTypeOptions'.
  Object literal may only specify known properties, and 'isAbstract' does not exist in type 'ObjectTypeOptions'.

7   isAbstract: true
```

**Comment by aerik2** [comment]
Fixed with this script:

```js
const replace = require("replace-in-file");
const results = replace.sync({
  files: "src/generated/graphql/**/*.ts",
  processor: (input) => input.replace(/isAbstract: true/g, ""),
  countMatches: true,
});
```

**Comment by MichalLytek** [comment]
That's the issue with breaking changes 😔

**Comment by aerykk** [comment]
> That's the issue with breaking changes 😔

What about throwing it behind a flag?

Also curious because the above PR are your changes, did it happen upstream somewhere?

**Comment by MichalLytek** [comment]
> What about throwing it behind a flag?

We can even try to read the TypeGraphQL version from project `package.json` and apply that conditional `isAbstract` generation automatically, so the flag would be used only when the detection mechanism failed.

**Comment by germanamz** [comment]
@MichalLytek is there something we can do to temporally solve this issue?

**Comment by germanamz** [comment]
I think this line here [type-class.ts:193](https://github.com/MichalLytek/typegraphql-prisma/blob/6784f3f9017efb117c3f45f117eb249ddf072743/src/generator/type-class.ts#LL193C32-L193C32) should be removed

**Comment by germanamz** [comment]
@MichalLytek I just created a PR to remove the flag from Model, Input and Output clases. https://github.com/MichalLytek/typegraphql-prisma/pull/393

**Comment by MichalLytek** [comment]
Closing as resolved by 19e841b 🔒
